morphic.js

```diff
@@ -8047,22 +8131,45 @@ StringMorph.prototype.deleteSelection = function () {
 };
 
 StringMorph.prototype.selectAll = function () {
+    var cursor;
     if (this.isEditable) {
         this.startMark = 0;
+        cursor = this.root().cursor;
```

Comment:
Hmm... See, I never get these things right. Everything I design ends up wanting to be accessed in (at least) both directions. Maybe we want to add a reference to the cursor to each editable element, so we don't have to do all this crawling. It might make some other things easier, as well. OTOH bidirectional references have a tendency to go out of synch and currupt. Need to think about this...